sys/src/cmd/frontusb/audio/audio.c

```diff
@@ -264,7 +264,7 @@ main(int argc, char *argv[])
 	createfile(fs.tree->root, "volume", user, 0666, nil);
 
 	snprint(buf, sizeof buf, "%d.audio", audiodev->id);
-	postsharesrv(&fs, nil, "usb", buf);
+	//postsharesrv(&fs, nil, "usb", buf);
```

Comment:
I am guessing we want to wait on this until devshr is in?

Reply:
yeah